modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -16,7 +16,7 @@ Restic is not supported in the OADP on ROSA with AWS STS environment. Ensure the
 
 .Prerequisites
 
-* A ROSA OpenShift Cluster with the required access and tokens. For instructions, see the procedure in _Preparing AWS credentials_.
+* A ROSA OpenShift Cluster with the required access and tokens. For instructions, see the procedure in _Preparing AWS credentials_. If you plan to use two different clusters for backing up and restoring, you need to prepare AWS credentials (including `ROLE_ARN`) for each cluster.
```

Comment:
```suggestion
* A {openshift-rosa} cluster with the required access and tokens. For instructions, see the procedure in _Preparing AWS credentials_. If you plan to use two different clusters for backing up and restoring, you need to prepare AWS credentials (including `ROLE_ARN`) for each cluster.
```

Where possible, use attributes for product names per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#product-title-and-version

Reply:
@abrennan89 
Done